package.json

```
@@ -2,7 +2,7 @@
   "name": "root",
   "private": true,
   "engines": {
-    "node": ">=20.1.0 <21"
+    "node": ">=20.1.0 <23"
```

I think we want something like this

```suggestion
    "node": ">=20.1.0 <21 || >=22 <23"
```